frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -76,3 +85,28 @@
     font-style: italic;
     padding: 0 0 0 2px;
 }
+
+.metadata .source-string-comment {
+    flex: 1 0 auto;
+}
+
+.metadata .source-string-comment .context-issue-button {
+    background: #333941;
+    border: 1px solid #3f4752;
+    border-radius: 4px;
+    color: #ccc;
+    float: right;
+    font-size: 11px;
+    font-style: normal;
+    font-weight: 100;
+    margin: -1px -1px 0 2px; /* Compensate for invisible border in default state */
```

Comment:
Sorry, could you please replace 2px with 4px to add some more spacing between text and the button?